descartes_core/CMakeLists.txt

```diff
@@ -3,13 +3,12 @@ project(descartes_core)
 
 add_definitions(-std=gnu++0x)
 find_package(catkin REQUIRED COMPONENTS
-  console_bridge
   moveit_core
   roscpp
+  cmake_modules
 )
 
 find_package(Boost REQUIRED)
-find_package(Eigen REQUIRED)
```

Comment:
Could be that you have Eigen installed in the default location (ie: `/usr/include`), which is always on the include path, or some other dependency already did a `find_package(Eigen ..)` somewhere.